### PR TITLE
feat: display links descriptions

### PR DESCRIPTION
**Pull request by richard015ar**
Issue: #410 

This PR allows rendering links within books' descriptions in the book card.

### Testing case
- Use `pressbooks_directory_e2e_by_last_updated` to ensure you can find a book with a link within the book description.
- Search by "Advanced Professional Communication", and you will find a book with a link in it.
- You should see a clickable link in the description.

To revisit: Styling work.

### CODE REVIEW

File: src/components/books/BookDetails.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@
     </div>
     <div data-cy="book-description">
       <vue-clamp
-        v-if="hasDescription"
+        v-if="hasDescription && item.description"
```
Comment (fdalcin) [inline src/components/books/BookDetails.vue]: Could you clarify why do we need this change in particular?

**Review by fdalcin — APPROVED** [review]
LGTM, tested locally and html tags are being rendered as expected. 

image
image

File: src/components/books/BookDetails.vue (hunk 1 above)
Comment (richard015ar) [reply src/components/books/BookDetails.vue]: good catch, we don't actually need it, I left it by accident. It was one of my exploratory changes. I'll remove it.

Comment (fdalcin) [reply src/components/books/BookDetails.vue]: Got it, thank you 👍🏻

**Pull request merged**